py/tests/unit/with_runtime_sparkling/test_gbm.py

```diff
@@ -44,3 +50,81 @@ def testLoadAndTrainMojo(prostateDataset):
     assert len(predMojo) == len(predModel)
     for i in range(0, len(predMojo)):
         assert predMojo[i] == predModel[i]
+
+
+@pytest.fixture(scope="module")
+def dataset(spark):
+    return spark \
+        .read.csv("file://" + unit_test_utils.locate("smalldata/insurance.csv"), header=True, inferSchema=True) \
+        .withColumn("Offset", log(col("Holders")))
+
+
+@pytest.fixture(scope="module")
+def gbmModelWithOffset(dataset):
+    gbm=H2OGBM(distribution="tweedie",
+               ntrees=600,
+               maxDepth=1,
+               minRows=1,
+               learnRate=0.1,
+               minSplitImprovement=0,
+               featuresCols=["District","Group","Age"],
+               labelCol="Claims",
+               offsetCol="Offset")
+    return gbm.fit(dataset)
+
+
+def testMOJOModelReturnsExpectedResultWhenOffsetColumnsIsSet(gbmModelWithOffset, dataset):
+    predictionCol = col("prediction")
+    predictionsDF = gbmModelWithOffset.transform(dataset)
+    detailsDF = predictionsDF.select(min(predictionCol).alias("min"),
+                                     max(predictionCol).alias("max"),
+                                     mean(predictionCol).alias("mean"))
+    result = detailsDF.first()
+    modelDetials = json.loads(gbmModelWithOffset.getModelDetails())
+
+    assert abs(-1.869702 - modelDetials['init_f']) < 1e-5, "Expected init_f to be {0}, but got {1}". \
```

Comment:
And other questions, just out of curiosity -> what init_f represents?

Reply:
http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/gbm.html#gbm-algorithm

`fk0` is beeing initialized to some constant, so I assume `init_f` represents the constant.